service_actor/test/resources/evolutions/test/1.sql

```diff
@@ -59,11 +59,11 @@ VALUES ('12345678-adb2-4b49-bce3-320ddfe6c90f', 'feide:fred@foo.com', 'Fred Flin
 INSERT INTO MUSARK_AUTH.USER_INFO (USER_UUID, SECONDARY_ID, NAME, EMAIL)
 VALUES ('d5a9a938-0905-470b-a892-88d0e7359579', 'feide:barney@foo.com', 'Barney Rubble', 'b.rubble@foo.bar.com');
 
-INSERT INTO MUSIT_MAPPING.ACTOR (ACTORNAME, DATAPORTEN_UUID, DATAPORTEN_USERNAME, APPLICATION_UUID)
-VALUES ('And, Arne1', null, 'andarn', '41ede78c-a6f6-4744-adad-02c25fb1c97c');
+INSERT INTO MUSIT_MAPPING.ACTOR (ACTORNAME, DATAPORTEN_UUID, DATAPORTEN_USERNAME, APPLICATION_UUID, MUSEUM_ID)
```

Comment:
Checked the `MUSIT_MAPPING.ACTOR` table just now, and there is indeed a column for museumId there. However, you'll need to change it's spelling.
But apparently it's not called `MUSEUM_ID` in that particular table, but `MUSEUMID`.